### PR TITLE
ZIOS-10243: Fix "Declined call" state

### PR DESCRIPTION
**Pull request by nicorsm**
### Issues

Calls rejected by the recipient should not be counted towards the amount of missed calls inside the conversations list.

### Solutions

Since we need this filter inside the conversations list only (system messages are still generated in the conversation), I've added a `isRelevant` field, in order to determine which notifications should be considered or not.

### CODE REVIEW

File: Resources/zmessaging.xcdatamodeld/zmessaging2.48.0.xcdatamodel/contents [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="14133" systemVersion="17C205" minimumToolsVersion="Xcode 4.3" sourceLanguage="Objective-C" userDefinedModelVersionIdentifier="2.48.0">
+<model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="14135" systemVersion="17C205" minimumToolsVersion="Xcode 4.3" sourceLanguage="Objective-C" userDefinedModelVersionIdentifier="2.48.0">
```
Comment (mikeger) [inline Resources/zmessaging.xcdatamodeld/zmessaging2.48.0.xcdatamodel/contents]: Please confirm the migration is not necessary.

Comment (nicorsm) [reply Resources/zmessaging.xcdatamodeld/zmessaging2.48.0.xcdatamodel/contents]: FYI I see that Vytis has released v2.49 of the data model (that's why tests are failing). I'll move the property there.

**Comment by typfel** [comment]
The name `isRelevant` is a bit unspecific I think. The property seems to similar to another property on messages `shouldGenerateUnreadDot`but as I understood it is about controlling weather it shows up in conversation list status row. Maybe a name like `relevantForConversationStatus` is more clear about what it is used for.

**Comment by nicorsm** [comment]
@zenkins release major

**Pull request merged**